test/runtests.jl

```diff
@@ -283,6 +283,14 @@ end
         @test !isnothing(i)
         @test is_no_escape(escapes.ssavalues[i])
     end
+
+    let
+        src, escapes = analyze_escapes((String,)) do s
+            b = isdefined(s, :value)
+            global bb = b
+        end
```

Comment:
`isdefined` here is just a builtin function call.
`:isdefind` expression will be introduced by `@isdefined` macro or other program constructs:
```julia
analyze_escapes((Ref{String},)) do a
    return @isdefined(b)
end
```

Reply:
Oh that's my miss. update it now